src/opm/parser/eclipse/EclipseState/NumericalAquifer.cpp

```diff
@@ -0,0 +1,327 @@
+/*
+  Copyright (C) 2020 Equinor ASA
+  Copyright (C) 2020 SINTEF Digital
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  ?You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <opm/parser/eclipse/Parser/ParserKeywords/A.hpp>
+
+#include <opm/parser/eclipse/EclipseState/NumericalAquifer.hpp>
+#include <opm/parser/eclipse/EclipseState/Aqucon.hpp>
+#include <opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.hpp>
+#include <opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp>
+#include <opm/parser/eclipse/EclipseState/Grid/TranCalculator.hpp>
+#include <opm/parser/eclipse/EclipseState/Grid/NNC.hpp>
+#include <opm/parser/eclipse/Deck/Deck.hpp>
+#include <opm/parser/eclipse/Deck/DeckRecord.hpp>
+#include <opm/common/OpmLog/OpmLog.hpp>
+
+#include <iostream>
+
+#include "AquiferHelpers.hpp"
+
+namespace Opm {
+
+NumericalAquifers::NumericalAquifers(const Deck& deck, const EclipseGrid& grid, const FieldPropsManager& field_props)
+{
+    using AQUNUM=ParserKeywords::AQUNUM;
+    if ( !deck.hasKeyword<AQUNUM>() ) return;
```

Comment:
Each man his syntax :-)

Reply:
This is the only situation that I do not add `{` for `if` statement.

Reply:
I think we have found a sort of truce when it comes to syntax; I just commented on the `return;` on the same line. It is fine - leave it!